main.go

```diff
@@ -65,6 +68,7 @@ func init() {
 	flag.StringVar(&inputFileName, "input-file", "-", "Input filename, use - for stdin")
 	flag.StringVar(&metadataFileName, "metadata-file", "-", "File to record banner-grab metadata, use - for stdout")
 	flag.StringVar(&logFileName, "log-file", "-", "File to log to, use - for stderr")
+	flag.StringVar(&prometheusAddress, "prometheus", "", "Address to use for prometheus server, value less than zero disables")
```

Comment:
What do you mean "value less than zero disables"? That's a confusing message. If it's disabled by default we might just keep it simple and exclude that. Second, what form is this supposed to be in? `127.0.0.1:8080`?